src/formatting.ts

```diff
@@ -23,20 +24,82 @@ export function serializeTag(
   return options.dataOnNextLine
     ? tags.reduce<string[]>(
       (lines, tag) =>
-        lines.concat([`@${tsdocTagName}`, `${tag.name} ${tag.description}`]),
+        lines.concat([
+          `@${tsdocTagName}${options.includeTypeInfo ? ` {${tag.type}}` : ""}`,
+          ...getMarkdownColumnLines(`${tag.name} ${tag.description}`, 80, [""]),
+        ]),
       [],
     )
-    : tags.map((tag) =>
-      `@${tsdocTagName} ${tag.name} ${tag.description}`.trim()
+    : tags.reduce<string[]>(
+      (lines, tag) =>
+        lines.concat(
+          getMarkdownColumnLines(
+            `@${tsdocTagName}${
+              options.includeTypeInfo ? ` {${tag.type}}` : ""
+            } ${tag.name.trim()} ${tag.description.trim()}`.trim(),
+            80,
+            [""],
+          ),
+        ),
+      [],
     );
 }
 
-export function formatMarkdown(md: string): string {
+export function getMarkdownColumnLines(
+  md: string,
+  maxWidth: number,
+  prefixes: string[],
+  rightPadChar?: string,
+  skipHangingPrefixes?: boolean,
+): string[] {
+  const prefixLength = prefixes[0].length;
+  const lastPrefix = prefixes[prefixes.length - 1];
+
+  return formatMarkdown(md, {
+    printWidth: maxWidth - prefixLength,
+  })
+    .split("\n")
+    .reduce<string[]>((tsdocLines, mdLine, i, all) => {
+      const tsdocLine = (prefixes[i] === undefined ? lastPrefix : prefixes[i]) +
+        mdLine;
+
+      // If rightPadChar is set, padRight to the expected width
+      tsdocLines.push(
+        rightPadChar === undefined || tsdocLine.length > maxWidth
+          ? tsdocLine
+          : tsdocLine + rightPadChar.repeat(maxWidth - tsdocLine.length),
+      );
+
+      if (
+        skipHangingPrefixes || prefixes.length <= all.length ||
+        i < all.length - 1
+      ) {
+        return tsdocLines;
+      }
+
+      // If the list of prefixes is longer than markdown lines, add additional lines until all
+      // prefixes are spent.
+
+      // If rightPadChar is set, padRight to the expected width
+      return tsdocLines.concat(
+        prefixes
+          .slice(all.length)
+          .map((p) =>
+            rightPadChar === undefined || p.length > maxWidth
+              ? p
+              : p + rightPadChar.repeat(maxWidth - p.length)
+          ),
+      );
+    }, []);
+}
+
+export function formatMarkdown(md: string, prettierOptions?: any): string {
```

Comment:
The type `any` isn't allowed and because of this the test are failing, please check it.